### PR TITLE
[MD][Cred mngment] - PoC List all credentials

### PR DESCRIPTION
**Pull request by noCharger**
### Description
* Add credentaion table under stack management
* List all credentials

Screen Shot 2022-07-04 at 6 59 05 PM

### Next step
1. Add creation page [WIP]
2. Submit form and route to '/api/credential_management/create' [WIP]

### CODE REVIEW

File: src/plugins/credential_management/server/crypto/crypto_material [hunk 1]
```diff
@@ -0,0 +1 @@
+{"keyName":"aes-name","keyNamespace":"aes-namespace","unencryptedMasterKey":{"type":"Buffer","data":[73,155,123,130,72,74,129,225,217,61,69,55,230,67,206,110,182,9,246,20,29,31,103,253,222,138,203,191,147,234,55,160]}}
```
Comment (seraphjiang) [inline src/plugins/credential_management/server/crypto/crypto_material:1]: if this is user defined config file , it should not be. put in src folder

File: src/plugins/credential_management/server/credential/credential_manager.ts [hunk 2]
```diff
@@ -11,27 +11,27 @@
 
 import { OpenSearchDashboardsRequest } from 'opensearch-dashboards/server';
 import { CryptoCli } from '../crypto';
+import { IBasicAuthCredentialMaterial, IAWSIAMCredentialMaterial } from '../../common';
+// interface Credential {
+//   readonly credential_name: string;
+//   readonly credential_type: string;
+//   readonly credential_material: BasicAuthCredentialMaterial | AWSIAMCredentialMaterial;
+// }
```
Comment (seraphjiang) [inline src/plugins/credential_management/server/credential/credential_manager.ts:19]: clean up code

File: src/plugins/credential_management/public/plugin.ts [hunk 3]
```diff
@@ -7,34 +9,73 @@ import {
...
+    // return {
+    //   getGreeting() {
+    //     return i18n.translate('credentialManagement.greetingText', {
+    //       defaultMessage: 'Hello from {name}!',
+    //       values: {
+    //         name: PLUGIN_NAME,
+    //       },
+    //     });
+    //   },
+    // };
```
Comment (seraphjiang) [inline src/plugins/credential_management/public/plugin.ts:78]: could we remove unused code

File: src/plugins/credential_management/public/components/utils.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import { SavedObjectsClientContract } from 'src/core/public';
+import { ICredential } from '../../common';
+// import { CredentialManagementStart } from '../plugin';
+
+export async function getCredentials(
+  savedObjectsClient: SavedObjectsClientContract,
+  defaultIndex: string
+  // credentialManagementStart: CredentialManagementStart
+) {
+  return (
```
Comment (seraphjiang) [inline src/plugins/credential_management/public/components/utils.ts:21]: use await to make code more readable and maintainable

**Review by seraphjiang — APPROVED** [review]

**Comment by noCharger** [comment]
Merged after clean up and resolved comments

**Pull request merged**